src/NServiceBus.Core/Licensing/LicenseManager.cs

```diff
@@ -63,38 +60,56 @@ static void LogFindResults(ActiveLicenseFindResult result)
             }
         }
 
-        void PromptUserForLicenseIfTrialHasExpired()
+        void OpenTrialExtensionPage()
         {
+            string url;
+
+            if (result.License.IsExtendedTrial)
+            {
+                url = "https://particular.net/extend-your-trial-45";
+            }
+            else
+            {
+                url = "https://particular.net/extend-nservicebus-trial";
+            }
+
             if (!(Debugger.IsAttached && Environment.UserInteractive))
             {
-                //We only prompt user if user is in debugging mode and we are running in interactive mode
+                Logger.WarnFormat("Go to '{0}' to extend your trial license", url);
```

Comment:
Would it be easier to copy-paste out of a log file if it was:

    Logger.WarnFormat("To extend your trial license, visit: {0}", url);

Beyond ease of copy-paste, it may eliminate some problems where a stray `'` at the end of a copied URL could cause an invalid URL or get blocked in some way?

There are other situations like that below but I won't mark up each one.